packages/eslint-config-4catalyzer/rules.js

```diff
@@ -23,7 +21,6 @@ module.exports = {
     },
   ],
   'no-plusplus': 'off',
-  'no-restricted-globals': ['error'].concat(restirctedGlobals),
```

Comment:
AFAICT this was just to allow `isNaN` and `isFinite`? But they both legitimately are sort of crappy.

Reply:
not sure i understand, this prevents you from using sketchy globals like `name` and `event` things that are likely local identifiers and you'd not get a "undef" warning for if you forgot to actually declare them in code

Reply:
no, i mean, this was already defined upstream: https://github.com/airbnb/javascript/blob/21b65e943c361f91af5a484460fb2c8d02e2ff92/packages/eslint-config-airbnb-base/rules/variables.js#L19

Reply:
o nice, 👍 didn't know that, just duplicated work then